src/build123d/build_common.py

```diff
@@ -1176,6 +1176,77 @@ def localize(cls, *points: VectorLike) -> Union[list[Vector], Vector]:
         return result
 
 
+def solids(select: Select = Select.ALL) -> ShapeList[Solid]:
+    """
+    Return Solids for the current builder context
+
+    Return either all or the solids created during the last operation.
```

Comment:
The `Select` enum actually has three values: ALL, LAST, and NEW.  There may be more options in the future so it's probably best to reword this line of the description to make it more generic.

Reply:
Likewise!